install/jaxr/docs/JAXRJavadocAssertions.html

```diff
@@ -24,7 +24,7 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java API for XML Registries<br>JAXR 1.0 - 1.0<br>
+<h2> Jakarta Registries 1.0 - 1.0<br>
```

Comment:
I don't know why "1.0" occurs twice.

Reply:
Seems to be something to do with the idea that these assertions could span multiple Spec versions. I'd just as soon leave this.